.github/workflows/publish.yml

```diff
@@ -69,85 +69,32 @@ jobs:
       - name: Upload zip
         uses: skx/github-action-publish-binaries@master
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN_ACCESS }}
```

Comment:
Do not change the name of the secret GITHUB_TOKEN as it is internal to github itself, so it is not necessary to configure it in the repository or orgranization.

https://docs.github.com/es/actions/security-guides/automatic-token-authentication